README.asciidoc

```diff
@@ -70,11 +70,26 @@ cargo build --release
 # replace `~/.local/bin/` with something on your `$PATH`
 ln -s $PWD/target/release/kak-lsp ~/.local/bin/
 
-mkdir -p ~/.config/kak-lsp 
+mkdir -p ~/.config/kak-lsp
 # or just link if you are okay with default config
 cp kak-lsp.toml ~/.config/kak-lsp/
 ----
 
+=== With plug.kak
+
+If you don't mind using plugin manager, you can install kak-lsp
+with https://github.com/andreyorst/plug.kak[plug.kak]. Add this code to your `kakrc`:
+
+---
+plug "ul/kak-lsp" do %{
+    cargo build --release
+    cargo install
```

Comment:
Consider using `make` (if not also `make install`), to let @ul have control of what is the proper way of compiling and installing the tool, to have room for future changes. For example, right now your command is not respecting `Cargo.lock`. There was some discussion about disabling LTO, maybe one day it would be possible to change from `$ make` to `$ make LTO=no`

Reply:
make has problems. Rust already has a build system builtin, and it's amazing. Well I guess I'll wait for @ul to response. If he will say to use make, I'll surely do, but for now let it stay this way. If the `--locked` parameter is the only difference I don't really see the problem here. Sure, what you're saying about make, and ability to silently change command is true, I just don't see why it can't be done with cargo alone. I've thought that there were some cargo configuration file which should handle such things, no?